### PR TITLE
schema: Add support for aquilon clusters

### LINKED ISSUE #71: Schema for aquilon cluster archetypes needed
Cluster archetypes in aquilon have a different profile structure to ordinary compilable "Quattor" archetypes, we should ship a schema for them.

### PR DESCRIPTION
**Pull request by jrha**
Resolves #71.

### CODE REVIEW

**Comment by jrha** [comment]
Could probably do with some squashing before being merged.

**Comment by jrha** [comment]
@hpcugentbot test this please

**Comment by ned21** [comment]
LGTM but agree some judicious squashing would not lose anything valuable.

**Comment by jrha** [comment]
Squashed to something more reasonable, should be good to go now.

**Comment by ned21** [comment]
@hpcugentbot retest this please

**Comment by ned21** [comment]
@hpcugentbot test this please

**Comment by stdweird** [comment]
@ned21 you'll have to wait for #170 if you want the tests to pass.

**Comment by jrha** [comment]
You'll just have to trust me that I've tested these changes :grin:

File: quattor/types/aquilon.pan [hunk 1]
```diff
@@ -77,6 +78,17 @@ type structure_maintenance = {
     "duration"   : long(1..100)
 };
 
+type structure_espinfo = {
+    "description"   ? string with check_esp_chars(SELF)
+    "class"         ? string with match(SELF, '(INFRASTRUCTURE|APPLICATION)')
+    # infrafunction is to give more detail to class for the
```
Comment (stdweird) [inline quattor/types/aquilon.pan]: annotation pls. (and ideally for all of them 😉 )

File: quattor/cluster_base.pan [hunk 2]
```diff
@@ -0,0 +1,7 @@
+unique template quattor/cluster_base;
```
Comment (stdweird) [inline quattor/cluster_base.pan]: prefixes typically indicate need for subtreem, in this case `quattor/cluster/base`

Comment (jrha) [reply quattor/cluster_base.pan]: So what do you suggest instead?

File: quattor/types/aquilon.pan (hunk 1 above)
Comment (jrha) [reply quattor/types/aquilon.pan]: I'm not sure I can provide too much of use, most of this came from MS as-is.

File: quattor/cluster_base.pan (hunk 2 above)
Comment (stdweird) [reply quattor/cluster_base.pan]: `quattor/cluster/base`

**Comment by stdweird** [comment]
@jrha can you have a look at the error in the test? it might be something form the test script, or an actual duplicate type.

**Comment by jrha** [comment]
Sure, meetings for now, will look later.

**Pull request merged**